src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -72,9 +72,9 @@ class PlannerPlanDetailsGetCommand extends GraphCommand {
     };
 
     return request
-      .get<{ value: [{ id: string, resourceProvisioningOptions: string[] }] }>(requestOptions)
+      .get<{ value: [{ id: string }] }>(requestOptions)
```

Comment:
Let's not define our own type inline but rather using typings from the `@microsoft/microsoft-graph-types` package

Reply:
done